internal/server/router/router.go

```diff
@@ -96,6 +96,39 @@ func (a *Router) IngestEvents(w http.ResponseWriter, r *http.Request, params api
 	a.config.IngestHandler.ServeHTTP(w, r, params)
 }
 
+func (a *Router) QueryEvents(w http.ResponseWriter, r *http.Request, params api.QueryEventsParams) {
+	logger := slog.With("operation", "queryEvents")
+
+	namespace := a.config.NamespaceManager.GetDefaultNamespace()
+	if params.NamespaceInput != nil {
+		namespace = *params.NamespaceInput
+	}
+
+	limit := 100
+	if params.Limit != nil {
+		limit = *params.Limit
```

Comment:
Does the OpenAPI validator verify that this is never larger than 100?

Reply:
It doesn't, good catch. I added validation.